src/constants/actionTypes.js

```diff
@@ -1,4 +1,14 @@
-export const INCREASE = 'INCREASE';
-export const DECREASE = 'DECREASE';
-export const INCREASE_ASYNC = 'INCREASE_ASYNC';
-// @todo: remove this kebab ^
+
+export const SETTINGS_CHANGE = 'SETTINGS_CHANGE'
+export const VIEW_DATE_CHANGE = 'VIEW_DATE_CHANGE'
+export const VIEW_FILTER_CHANGE = 'VIEW_FILTER_CHANGE'
+export const EVENTS_LOAD_STARTED = 'EVENTS_LOAD_STARTED'
+export const EVENTS_LOAD_COMPLETED = 'EVENTS_LOAD_COMPLETED'
+export const EVENTS_LOAD_FAILED = 'EVENTS_LOAD_FAILED'
+export const SIDEBAR_DISPLAY = 'SIDEBAR_DISPLAY'
+export const EVENT_DISPLAY = 'EVENT_DISPLAY'
+export const RESET_DISPLAY = 'RESET_DISPLAY'
+export const SEARCH_REQUEST = 'SEARCH_REQUEST'
+export const SEARCH_RESPONSE = 'SEARCH_RESPONSE'
+export const SEMESTER_LOAD_COMPLETED = 'SEMESTER_LOAD_COMPLETED'
+export const CLIENT_CHANGE = 'CLIENT_CHANGE'
```

Comment:
![Amount of boilerplate in this file is too damn high!](https://i.imgflip.com/qyif2.jpg)

#justsaying